lib/rules/web/admin_console/4.8/operator_hub.xyaml

```diff
@@ -383,12 +383,12 @@ wait_filter_section_loaded:
 choose_one_checkbox:
   element:
     selector:
-      xpath: //input[contains(@class, 'pf-c-check__input') and @title='<checkbox_text>']
+      xpath: //input[contains(@class, 'pf-c-check__input') and contains(@title,'<checkbox_text>')]
     op: click
 check_label_and_value_in_modal:
   element:
     selector:
-      xpath: //h5[contains(@class,'properties-side-panel-pf-property-label') and text()='<label>']/../div[contains(@class,'properties-side-panel-pf-property-value')]/div[text()='<value>']
+      xpath: //h5[contains(@class,'properties-side-panel-pf-property-label') and text()='<label>']/../div[contains(@class,'properties-side-panel-pf-property-value')]/div[contains(text(),'<value>')]
```

Comment:
How about also update to use "contains" for the label text?

Reply:
Yeah I agree @yanpzhan  let me update

Reply:
Please be careful with the contais method @yanpzhan and @yapei , if there are some similar labels, it may cause unexpected issue. In this case, all labels are different, so don't worry.

Reply:
yeah, we only use `contains` in very certain conditions @JianLi-RH